### DISCUSSION TITLE
ChatClaude To Termux /Original Author By InterestingDarkness

### DISCUSSION THREAD

**Discussion by viopsa233** [viopsa233/AI-testers-Forum#74] [技术交流]
基本步骤和[ChatSydney](https://github.com/viopsa233/AI-testers-Forum/discussions/44)一样,这里就不解释了
然后到创建软链接之前,先获取你的User OAuth Token,怎么弄看[这个](https://mp.weixin.qq.com/s?__biz=Mzg4MjkzMzc1Mg==&mid=2247483961&idx=1&sn=c009f4ea28287daeaa4de17278c8228e#:~:text=%E7%AA%97%E5%8F%A3%E5%BC%80%E5%A7%8B%E4%BD%BF%E7%94%A8-,Slack%20API,-%E5%BD%93%E7%84%B6%E8%BF%99%E4%BA%9B%E5%86%85%E5%AE%B9)
**在添加[User Token Scopes](https://mp.weixin.qq.com/s?__biz=Mzg4MjkzMzc1Mg==&mid=2247483961&idx=1&sn=c009f4ea28287daeaa4de17278c8228e#:~:text=%E7%9A%84%20User%20Token-,%E6%B7%BB%E5%8A%A0%20Scope,-%E3%80%82%C2%A0)记得加一个groups:history!**

然后打开claude.json,把你的slackUserTokend,claudeMemberId,slackChannelId填进去

- 其中,claudeMemberId是指你在应用和Claude聊天的成员ID
![网页捕获_16-6-2023_214714_app slack com (1)](https://github.com/viopsa233/AI-testers-Forum/assets/118115208/2a98819c-04ec-4675-bbcc-89a56aa2a62a)

- slackUserToken是你的app的Token
![网页捕获_16-6-2023_215225_api slack com (1)](https://github.com/viopsa233/AI-testers-Forum/assets/118115208/346ddc47-cf4f-4135-8411-76cba764f7ef)

- slackChannelId是你的公开频道的ID(**不是你在应用和Claude聊天的ID!!!**)
![网页捕获_16-6-2023_215054_app slack com (1)](https://github.com/viopsa233/AI-testers-Forum/assets/118115208/8eab0b68-7d47-4787-aa6d-53da484850b1)

**开启步骤和[ChatSydney](https://github.com/viopsa233/AI-testers-Forum/discussions/44)也一样**

然后,ChatSydney,启动!

选择Claude就可以聊天了
![Screenshot_2023-06-16-21-38-13-26](https://github.com/viopsa233/AI-testers-Forum/assets/118115208/d3edaef5-62fa-4afd-8bea-213cf84bba87)

**Comment by gunshipwu** [comment]
InterestingDarkness 大佬的贴子和项目怎么都不见了....有人知道他怎么了吗？

**Reply by viopsa233** [reply]
不知道,之前也这样
